src/main/java/com/aws/greengrass/deployment/bootstrap/BootstrapManager.java

```diff
@@ -316,17 +325,16 @@ private boolean defaultRunWithChanged(Map<String, Object> newNucleusParameters,
         if (Utils.stringHasChanged(Coerce.toString(currentValues.get(RUN_WITH_DEFAULT_POSIX_USER)),
                 Coerce.toString(runWithDefault.get(RUN_WITH_DEFAULT_POSIX_USER)))) {
             logger.atInfo().kv(RUN_WITH_TOPIC + "." + RUN_WITH_DEFAULT_POSIX_USER,
-                    runWithDefault.get(RUN_WITH_DEFAULT_POSIX_USER))
-                    .log(RESTART_REQUIRED_MESSAGE);
+                    runWithDefault.get(RUN_WITH_DEFAULT_POSIX_USER)).log(RESTART_REQUIRED_MESSAGE);
             changed = true;
         }
-        if (Utils.stringHasChanged(Coerce.toString(currentValues.getOrDefault(RUN_WITH_DEFAULT_POSIX_SHELL,
-                RUN_WITH_DEFAULT_POSIX_SHELL_VALUE)),
+        if (Utils.stringHasChanged(
+                Coerce.toString(
+                        currentValues.getOrDefault(RUN_WITH_DEFAULT_POSIX_SHELL, RUN_WITH_DEFAULT_POSIX_SHELL_VALUE)),
                 Coerce.toString(runWithDefault.getOrDefault(RUN_WITH_DEFAULT_POSIX_SHELL,
                         RUN_WITH_DEFAULT_POSIX_SHELL_VALUE)))) {
             logger.atInfo().kv(RUN_WITH_TOPIC + "." + RUN_WITH_DEFAULT_POSIX_SHELL,
-                    runWithDefault.get(RUN_WITH_DEFAULT_POSIX_SHELL))
-                    .log(RESTART_REQUIRED_MESSAGE);
+                    runWithDefault.get(RUN_WITH_DEFAULT_POSIX_SHELL)).log(RESTART_REQUIRED_MESSAGE);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Untrusted input is passed to a method that executes arbitrary code, creating a code injection vulnerability. This allows attackers to execute malicious code within the application, potentially leading to data theft or system compromise. Validate input using allowlists with `Arrays.asList().contains()`, use regex matching with `String.matches()`, or parse input safely with `Integer.parseInt()` or `Boolean.parseBoolean()`. [Learn more](https://owasp.org/www-community/attacks/Code_Injection)